.github/scripts/spellcheck.words

```diff
@@ -171,6 +171,7 @@ doxygen
 drftpd
 dsa
 Dudka
+dweb
```

Comment:
This does not seem right. It is not a word to accept I think. You should probably rather quote this correctly in the documentation.

Reply:
Reverted. I think adding backticks around all urls (not all had it) solved the reason for adding this.